docs/samples/calling/app.js

```diff
@@ -163,32 +163,43 @@ function changeEnv() {
   enableProduction.innerHTML = enableProd ? 'In Production' : 'In Integration';
 }
 
-const guestUrl = 'https://webexapis.com/v1/guests/token';
-const guestIssuerAccessToken = '';
-
-// Guest access token via Service App
-async function getGuestAccessToken() {
-  const response = await fetch(guestUrl, {
-    method: 'post',
-    headers: {
-      Accept: 'application/json',
-      'Content-Type': 'application/json',
-      Authorization: `Bearer ${guestIssuerAccessToken}`,
-    },
-    body: JSON.stringify({
-      subject: 'Guest token for Webex Calling SDK Sample App',
-      displayName: 'Calling Guest User',
-    }),
-  });
+// Note: Keeping the below code for developer reference
```

Comment:
If this code is not needed anymore, please remove it

Reply:
Done